collect_app/src/test/java/org/odk/collect/android/entities/EntitiesRepositoryTest.kt

```diff
@@ -63,32 +63,13 @@ abstract class EntitiesRepositoryTest {
         val wine = Entity("wines", "1", "Léoville Barton 2008", version = 1)
         repository.save(wine)
 
-        val updatedWine = wine.copy(label = "Léoville Barton 2009")
+        val updatedWine = wine.copy(label = "Léoville Barton 2009", version = 1)
         repository.save(updatedWine)
 
         val wines = repository.getEntities("wines")
         assertThat(wines, contains(updatedWine))
     }
 
-    @Test
-    fun `#save does not update existing entity with matching id but not list`() {
```

Comment:
Why is this testpassing (would be if it still existed) if entities should have unique id? Do we have a test for that scenario?

Reply:
I think when I initially added this test I was thinking that two entities in different lists could have the same ID. This is actually not something we want to support though (entities should have unique IDs across lists), so I removed the test so as not to imply the opposite. I'll add a test to specify what happens if you pass `save` an entity with an existing ID in a different list.